test/Stats.h

```diff
@@ -21,6 +21,7 @@
 #include <vector>
 
 #include "TestHelper.h"
+//Execution stats (time/gas usage/OOG exceptions) for vm tests and tests that are using the vm.
```

Comment:
about Stats.h and what does it do.  There were complaints about code lacking comment coverage.